packages/react-next/src/components/Popup/Popup.types.ts

```diff
@@ -51,5 +51,9 @@ export interface IPopupProps extends React.HTMLAttributes<HTMLDivElement> {
    * contains focus. If this is provided, focus will not be restored automatically,
    * you'll need to call originalElement.focus()
    */
-  onRestoreFocus?: (options: { originalElement?: HTMLElement | Window; containsFocus: boolean }) => void;
+  onRestoreFocus?: (options: {
+    originalElement?: HTMLElement | Window;
+    containsFocus: boolean;
+    documentContainsFocus?: boolean;
```

Comment:
I don't believe that this should be able to be undefined. If we can't determine if a document has focus, let's set it to false.

Reply:
Would this be a breaking change for the current api?

Reply:
Or back compat is mainly relevant for office-ui-fabric-react? Here, I believe I can make it a required without any such concerns